sdk/tests/evm/native/balance.spec.ts

```
@@ -0,0 +1,65 @@
+import { evm } from "../../../src/";
+import { accountsEvm, blockchainEvm as blockchain } from "../../mocks/contract";
+
+describe("Test token allowance request", () => {
+  beforeEach(async () => await resetMocks());
+  beforeEach(
+    async () =>
+      await mock({
+        blockchain,
+        accounts: { return: accountsEvm },
+      })
+  );
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)